growth/too/tasks/tiles.py

```diff
@@ -198,6 +198,7 @@ def params_struct(dateobs, tobs=None, filt=['r'], exposuretimes=[60.0],
     params["mindiff"] = mindiff
 
     params = gwemopt.segments.get_telescope_segments(params)
+    params = gwemopt.utils.params_checker(params)
```

Comment:
What does `params_checker` do?

Reply:
It chooses sensible defaults for parameters that are not uniquely specified.

Reply:
Why is this a new function? Why don't the existing functions provide sensible defaults?

Reply:
In general, they do, with a few exceptions that arise from the command line version of the tool setting them. This helps rectify this.